spot_wrapper/cam_wrapper.py

```diff
@@ -795,10 +795,10 @@ def set_ptz_position(self, ptz_name, pan, tilt, zoom, blocking=False):
                 self._get_ptz_description(ptz_name)
             )
             while (
-                datetime.datetime.now() - start_time < datetime.timedelta(seconds=3)
-                or not math.isclose(current_position.pan, pan, abs_tol=1)
-                or not math.isclose(current_position.tilt, tilt, abs_tol=1)
-                or not math.isclose(current_position.zoom, zoom, abs_tol=0.5)
+                (not math.isclose(current_position.pan.value, pan, abs_tol=1)
```

Comment:
As slight nit (but moreso to keep with the accepted code style), can you refactor the logic as
```
while (
    not (math.isclose(current_position.pan.value, pan, abs_tol=1) 
    and math.isclose(current_position.tilt.value, tilt, abs_tol=1)
    and math.isclose(current_position.zoom.value, zoom, abs_tol=0.5))
    and datetime.datetime.now() - start_time < datetime.timedelta(seconds=3)
```

I believe that logic is a bit easier to read and equivalent. This way, it's more obvious that we are exiting only in the case where each PTZ approaches the correct setting and the clock is under the convergence time.

Reply:
Sure, I agree. Potentially we could also extract the timeout and put it as a conditional break inside the while loop body.